### DISCUSSION TITLE
Rector for Symfony Contributions

### DISCUSSION THREAD

**Discussion by zairigimad** [symfony/symfony#43590] [Q&A]
Hello,

While contributing to Symfony, sometimes we need to fix some coding standards, with the fabbot is very easy to do, but I guess that can be done automatically with Rector in GitHub action, without doing anything in local and push the coding standards.

what do you think about using Rector to fix coding standards automatically?
Thanks

**Comment by stof** [comment]
We rely on php-cs-fixer to fix the coding standards (that's the tool that fabbot uses as well for its `coding standards` section, so it won't report things that are not fixable by that tool).

However, we don't want to have a `Fix coding standards` commit in the git history after each merge commit, which is what would need to happen if we don't require pull requests to respect the coding standards.
